pkgs/by-name/sd/SDL1/package.nix

```diff
@@ -143,13 +133,41 @@ stdenv.mkDerivation (finalAttrs: {
 
   setupHook = ./setup-hook.sh;
 
-  passthru = { inherit openglSupport; };
+  passthru = {
+    inherit openglSupport;
+
+    tests = lib.filterAttrs (key: value: value.meta.available) {
+      inherit
+        blender
+        directfb
+        hheretic
+        hhexen
+        mednafen
+        onscripter-en
+        powermanga
+        SDL_Pango
+        SDL_gfx
+        SDL_image
+        SDL_mixer
+        SDL_net
+        SDL_sound
+        SDL_ttf
+        sfxr
+        vlc
+        ;
+    };
```

Comment:
this passthru test list isn't very meaningful since #389106 was merged. I am not even sure this is a good idea to have at all, hydra and nixpkgs-review should catch builds of any dependents.